src/popup/Settings.tsx

```diff
@@ -334,6 +373,9 @@ export function Settings(props: IPopupProps) {
                         <FormItem required name='apiURL' label='API URL'>
                             <Input size='compact' onBlur={onBlur} />
                         </FormItem>
+                        <FormItem required name='apiURLPath' label='API URL Path'>
```

Comment:
Can apiURLPath be automatically generated based on the provider? This way users don't have to input it, reducing mental burden.

Reply:
The rule is `/openai/deployments/{deployment-id}/completions?api-version={api-version}`
ref: https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference#completions

There are two variables here.

At the same time, I also consider that if users deploy their own Proxy, it will be very convenient to define Path by themselves.

Reply:
@tao12345666333 make sense